### DISCUSSION TITLE
gamma ray binary analysis with gammapy

### DISCUSSION THREAD

**Discussion by biswaraj7991** [gammapy/gammapy#3696] [Help] [answered]
My query : Is there any data for gamma ray binaries (like LS5039 or LSI61303) in the datastore of gammpy ?

**Comment by registerrier** [comment, marked as answer]
Hello @biswaraj7991, the data shipped with Gammapy contain a few observations from the CTA DC1 (GC),  a few Crab observations from various instruments and the HESS DL3 DR1, which does not contain any observations of a binary.
You might look into the internal CTA-consortium pages for DC1 or GPS data.

**Reply by biswaraj7991** [reply]
Okay , Thanks
